src/main/java/mate/jdbc/service/CarServiceImpl.java

```diff
@@ -0,0 +1,63 @@
+package mate.jdbc.service;
+
+import java.util.ArrayList;
+import java.util.List;
+import mate.jdbc.dao.CarDao;
+import mate.jdbc.lib.Inject;
+import mate.jdbc.lib.Service;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+
+@Service
+public class CarServiceImpl implements CarService {
+    @Inject
+    private CarDao carDao;
+
+    @Override
+    public Car create(Car car) {
+        return carDao.create(car);
+    }
+
+    @Override
+    public Car get(Long id) {
+        return carDao.get(id).get();
+    }
+
+    @Override
+    public List<Car> getAll() {
+        return carDao.getAll();
+    }
+
+    @Override
+    public Car update(Car car) {
+        return carDao.update(car);
+    }
+
+    @Override
+    public boolean delete(Long id) {
+        return carDao.delete(id);
+    }
+
+    @Override
+    public void addDriverToCar(Driver driver, Car car) {
+        if (car.getDrivers() != null) {
+            car.getDrivers().add(driver);
+        } else {

```

Comment:
do we need this?

Reply:
we can run without if-else statement (if you have meant that) but I have no idea how we can run code without null-checking: https://prnt.sc/1s1ofvf, https://prnt.sc/1s1oi1x